pkg/proc/target_exec.go

```diff
@@ -213,7 +214,18 @@ func (grp *TargetGroup) Continue() error {
 					}
 					return grp.StepInstruction(false)
 				}
-			} else {
+			case curbp.SteppingIntoRangeOverFuncBody:
+				if err := conditionErrors(grp); err != nil {
+					return err
+				}
+				if err := dbp.ClearSteppingBreakpoints(); err != nil {
+					return err
+				}
+				if err := next(dbp, false, false); err != nil {
+					return err
+				}
+				//continue
```

Comment:
Leftover?

Reply:
Actually, no. I meant to say that target's execution continues here. I've expanded it.